R/RunDiagnostics.R

```diff
@@ -123,6 +123,10 @@ runCohortDiagnostics <- function(packageName = NULL,
                                  incremental = FALSE,
                                  incrementalFolder = file.path(exportFolder, "incremental")) {
   
+  if (runTimeSeries) {
```

Comment:
This is incorrect; deprecation means that it will be removed in a future version. Here it's just being removed (which is fine) but the `runTimeSeries` parameter should probably just be removed and we can add this info to the release notes.

Reply:
i see - so if we incorporate these changes in future release and a user has a legacy code - it would throw an error. 

is that acceptable?

Reply:
We're replacing `runCohortDiagnostics` with `executeDiagnostics` anyway; I think we should just remove the parameter; adding it to the release notes will clear up any confusion.